eframe/src/web/input.rs

```diff
@@ -14,6 +14,8 @@ pub fn button_from_mouse_event(event: &web_sys::MouseEvent) -> Option<egui::Poin
         0 => Some(egui::PointerButton::Primary),
         1 => Some(egui::PointerButton::Middle),
         2 => Some(egui::PointerButton::Secondary),
+        3 => Some(egui::PointerButton::Extra1),
+        4 => Some(egui::PointerButton::Extra2),
```

Comment:
Seems a bit weird to begin indexing at `1`, but perhaps these are standard names?

Reply:
Sorry I didn't come up with better names. I can rename them to Forward & Backward but I don't know if it's meaningful enough because they go by these names mostly in browsers.

Reply:
Let's keep it as is